### PR TITLE
Merge `develop` into `master`

### PR DESCRIPTION
**Pull request by ad-daniel**
Conflicts:
   - tests/expected/RobotWithDummyLink.proto
   - tests/test_export.py
   - urdf2webots/importer.py
   - urdf2webots/parserURDF.py
   - urdf2webots/writeRobot.py

### CODE REVIEW

**Comment by ad-daniel** [comment]
This PR can now be reviewed

File: tests/expected/HumanR2022a.proto [hunk 1]
```diff
@@ -0,0 +1,939 @@
+#VRML_SIM R2022a utf8
+# license: Apache License 2.0
+# license url: http://www.apache.org/licenses/LICENSE-2.0
+# This is a proto file for Webots for the HumanR2022a
+# Extracted from: /home/benja/urdf2webots/tests/sources/gait2392_simbody/urdf/human.urdf
```
Comment (BenjaminDeleze) [inline tests/expected/HumanR2022a.proto:5]:
```suggestion
# Extracted from: /home/benja/urdf2webots/tests/sources/gait2392_simbody/urdf/human.urdf
```
Why it contains a path on my computer ? I do not remember even pushing stuff on this repo other than version number update.

File: tests/test_export.py [hunk 2]
```diff
@@ -8,27 +8,34 @@
 
 from urdf2webots.importer import convertUrdfContent, convertUrdfFile
 
-testDirectory = os.path.abspath(os.path.dirname(os.path.realpath(__file__)))
+rootDirectory = os.path.dirname(os.path.dirname(__file__)) + os.path.sep
```
Comment (omichel) [inline tests/test_export.py:11]: I would remove the `+ os.path.sep` to be consistent with other `*Directory` variables and add `+ os.path.sep` in the `replace` statement where `rootDirectory` is used or define a new variable `rootDirectoryWithFinalSeparator`.

**Review by omichel — CHANGES_REQUESTED** [review]

**Review by omichel — APPROVED** [review]
Thank you.

**Pull request merged**